node/src/manager/commands/rewind.rs

```diff
@@ -104,30 +106,19 @@ pub async fn run(
     };
 
     println!("Pausing deployments");
-    let mut paused = false;
     for deployment in &deployments {
-        if let Some(node) = &deployment.node_id {
-            if !node.starts_with(PAUSED) {
-                let loc = deployment.locator();
-                let node =
-                    NodeId::new(format!("{}{}", PAUSED, node)).expect("paused_ node id is valid");
-                subgraph_store.reassign_subgraph(&loc, &node)?;
-                println!("  ... paused {}", loc);
-                paused = true;
-            }
-        }
-    }
+        let paused = pause_or_resume(primary.clone(), &sender, &deployment.locator(), true);
 
-    if paused {
-        // There's no good way to tell that a subgraph has in fact stopped
-        // indexing. We sleep and hope for the best.
-        println!(
-            "\nWaiting {}s to make sure pausing was processed",
-            sleep.as_secs()
-        );
-        thread::sleep(sleep);
+        if paused.is_ok() {
```

Comment:
is_ok() doesn't mean the subgraph is paused, it only means the result is the Ok variant. 

We want to check the boolean value of paused, So handle the result from `pause_or_resume` with a `?` and then do 
``` if paused {}```